hazelcast/include/hazelcast/util/BlockingConcurrentQueue.h

```diff
@@ -37,7 +37,7 @@ namespace hazelcast {
             BlockingConcurrentQueue(size_t max_queue_capacity) : capacity_(max_queue_capacity), is_interrupted_(false) {
```

Comment:
It looks like this class is not used anymore.

Reply:
Removed it and related tests.